packages/api/endpoints/granules.js

```diff
@@ -88,9 +88,6 @@ const create = async (req, res) => {
 
   try {
     await createGranuleFromApi(granule, knex, esClient);
-    if (inTestMode()) {
```

Comment:
Can we check to see if `addToLocalES` is used elsewhere and if not delete it altogether? I don't want to keep unnecessary code around

Reply:
[`f2eef77` (#2589)](https://github.com/nasa/cumulus/pull/2589/commits/f2eef77cba67a1c09d5fbd38139fb7d850aa7d7e)